README.md

```diff
@@ -292,10 +292,6 @@ Alternative build without Makefile:
 ```bash
 sphinx-build -b html <sourcedir> <builddir>
 ```
-PDF output:
```

Comment:
shall we maybe instead just leave a sentence to say something like _Generating a PDF version of docs requires installation of extra `tex` tools._
?

Reply:
ok, let's leave it out